ocs_ci/deployment/ibm_deployment.py

```diff
@@ -0,0 +1,1021 @@
+
+            sc = helpers.default_storage_class(interface_type=constants.CEPHBLOCKPOOL)
+
+            # Get the list of monitoring pods
+            pods_list = get_all_pods(
+                namespace=defaults.OCS_MONITORING_NAMESPACE,
+                selector=['prometheus', 'alertmanager']
+            )
+
+            # Create configmap cluster-monitoring-config and reconfigure
+            # storage class and telemeter server (if the url is specified in a
+            # config file)
+            create_configmap_cluster_monitoring_pod(
+                sc_name=sc.name,
+                telemeter_server_url=config.ENV_DATA.get("telemeter_server_url"))
+
+            # Take some time to respin the pod
+            waiting_time = 45
+            logger.info(f"Waiting {waiting_time} seconds...")
+            time.sleep(waiting_time)
+
+            # Validate the pods are respinned and in running state
+            retry(
+                (CommandFailed, ResourceWrongStatusException),
+                tries=3,
+                delay=15)(
+                validate_pods_are_respinned_and_running_state
+            )(
+                pods_list
+            )
+
+            # Validate the pvc is created on monitoring pods
+            validate_pvc_created_and_bound_on_monitoring_pods()
+
+            # Validate the pvc are mounted on pods
+            retry((CommandFailed, AssertionError), tries=3, delay=15)(
+                validate_pvc_are_mounted_on_monitoring_pods
+            )(pods_list)
+        elif config.ENV_DATA.get('monitoring_enabled') and config.ENV_DATA.get("telemeter_server_url"):
+            # Create configmap cluster-monitoring-config to reconfigure
+            # telemeter server url when 'persistent-monitoring' is False
+            create_configmap_cluster_monitoring_pod(
+                telemeter_server_url=config.ENV_DATA["telemeter_server_url"])
+
+        # Change registry backend to OCS CEPHFS RWX PVC
+        registry.change_registry_backend_to_ocs()
+
+        # Verify health of ceph cluster
+        # TODO: move destroy cluster logic to new CLI usage pattern?
+        logger.info("Done creating rook resources, waiting for HEALTH_OK")
+        try:
+            ceph_health_check(namespace=self.namespace, tries=30, delay=10)
+        except CephHealthException as ex:
+            err = str(ex)
+            logger.warning(f"Ceph health check failed with {err}")
+            if "clock skew detected" in err:
+                logger.info(
+                    f"Changing NTP on compute nodes to"
+                    f" {constants.RH_NTP_CLOCK}"
+                )
+                update_ntp_compute_nodes()
+                assert ceph_health_check(
+                    namespace=self.namespace, tries=60, delay=10
+                )
+
+        # patch gp2/thin storage class as 'non-default'
+        self.patch_default_sc_to_non_default()
+
+        # Modify Noobaa endpoint auto scale values according to the cluster specs
```

Comment:
I think that there are some core duplicity with ocs_ci/deployment/deployment content, like the section here

Reply:
I removed redundant code. ibm.py has only whats used in it.